include/inviwo/core/datastructures/datamapper.h

```diff
@@ -54,36 +55,87 @@ class IVW_CORE_API DataMapper {
     DataMapper(const DataMapper& rhs);
     DataMapper& operator=(const DataMapper& that);
 
-    dvec2 dataRange;        ///< Minimum and maximum data range
-    dvec2 valueRange;       ///< Minimum and maximum value range
-    std::string valueUnit;  ///< Unit, i.e. Hounsfield/absorption/W.
+    /**
+     * The ´dataRange´ is used to normalize the "raw" values to [0,1]
+     * Typically the minimum and maximum of the "raw" data is used. For 8 and 16 integer data that
+     * is often the same or close to the same as the minimum and maximum of the data type. For data
+     * where the minimum and maximum is far from the minimum and maximum of the data type, the
+     * minimum and maximum has to either be calculated from the data, read from the input, or
+     * entered by the user.
+     */
+    dvec2 dataRange;
+
+    /**
+     * The `valueRange` corresponds to the "true" values in the "raw" data. The "raw" data will
+     * first be normalized to [0,1] using the `dataRange` and mapped to the "true" range using the
+     * `valueRange`. An example might be helpful: if water temperature values in the range of
+     * [0.0, 30.0] °C are stored as 8 bit unsigned int, using the full data range from [0,255]. Then
+     * the `dataRange` would be [0, 255] and the `valueRange` [0.0, 30.0].
+     * Using the same values for `dataRange` and `valueRange` means that the original "raw" values
+     * will be used.
+     */
+    dvec2 valueRange;
+
+    /**
+     * The `valueAxis` holds the `name` and `unit` of the quantity. Following the example above
+     * the `name` would be "water temperature" and `unit` "°C".
+     */
+    Axis valueAxis;  ///< Name and Unit, i.e. "absorption", "Hounsfield".
```

Comment:
Not sure this belongs here as it adds extra responsibility to this class. Sure, it is related but also quite hidden.
It would be more visible if we put it next to the axis units of the Image/Volume as we have done for the Column. Would that make more sense?